format-specs/compatible-parquet.md

```diff
@@ -0,0 +1,41 @@
+
+## Compatibility Guidelines
+
+The core idea of the compatibility guidelines is to have the output match the defaults of the official spec as closely as possible, so it is very easy for tools to simply add the appropriate Parquet metadata and create valid GeoParquet. The guidelines are as follows:
+
+* The geometry column should be named 'geometry'.
+
+* The geometry column should be a `BYTE_ARRAY` with Well Known Binary (WKB) used to define the geometries, as defined in the [encoding](./geoparquet.md#encoding) section of the GeoParquet spec.
+
+* All data is stored in longitude, latitude based on the WGS84 datum, as defined as the default in the [crs](./geoparquet.md#crs) section of the GeoParquet spec.
+
+### Data Reader Assumptions
+
+The above are the key recommendations a data producer should follow. Any implemented reader will need to make the following assumptions when reading one of these columns, unless the user supplies additional information that they are aware of:
+
+* The geometry_types values is an empty array, signaling the geometry type is not known and the reader should make no assumptions about the types, as defined in the [geometry_types](./geoparquet.md#geometry_types) section of the spec.
+
+* Any CRS-aware reader should assume that the CRS is OGC:CRS84 as explained in the [crs](./geoparquet.md#crs) section of the spec. (Or it could assume it is EPSG:4326 but overriding the axis order to assume longitude latitude as explained in the [Coordinate axis order](./geoparquet.md#coordinate-axis-order)) section.
+
+* No assertions are made on the winding order, the default of the [orientation](./geoparquet.md#orientation) section of the spec.
+
+* Edges are planar, as explained to be the default in the [edges](./geoparquet.md#edges) section of the spec.
+
+## Data Reader Implementation Considerations
+
+Reading this non-compliant geospatial data from Parquet should ideally work with no user intervention if the producer followed all the guidelines and named their geometry column 'geometry'. Readers can optionally support user input (in whatever manner works for the tool / library) to provide hints for metadata that is not inline with the lowest common denominator compatibility. This would include things like letting the user supply a geometry column name (ie something other than 'geometry'), using Well Known Text (WKT) in a `STRING` column instead of WKB, providing a more specific geometry_type value, or providing other enhanced metadata (specifying the CRS, the winding order, the edges, the bbox or the epoch).
```

Comment:
Assuming that `STRING` geometry columns contain WKT is a clever fallback.

Reply:
That was @tschaub's suggestion I'm pretty sure.